### PR TITLE
S9 pagination

### PR DESCRIPTION
**Pull request by lingsat**

### CODE REVIEW

File: src/features/products/components/ListPagination/ListPagination.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+              }}
+            />
+          )}
+        />
+      </div>
+      <Button
+        text="Show more products"
+        arrowDirection="bottom"
+        onCLick={handleIncreaseProdPerPage}
+        disabled={activePagesArr.includes(pageCount) || !pageCount}
```
Comment (Yuliia143) [inline src/features/products/components/ListPagination/ListPagination.tsx]: Logic

File: src/features/products/types/pagination.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+export interface IPaginationData {
+  pageCount: number;
+  activePagesArr: number[];
+  lastActivePage: number;
+}
+
+export enum EPagination {
+  INITIAL__PAGE = 1,
+  PAGES__PANGE = 3,
```
Comment (Yuliia143) [inline src/features/products/types/pagination.ts]: *Range

**Review by Yuliia143 — APPROVED** [review]

File: src/features/products/types/pagination.ts (hunk 2 above)
Comment (lingsat) [reply src/features/products/types/pagination.ts]: Remove "PAGES__RANGE" from enum - I used it before in react-paginate library, but then I swap to MUI Pagination

File: src/features/products/components/ListPagination/ListPagination.tsx (hunk 1 above)
Comment (lingsat) [reply src/features/products/components/ListPagination/ListPagination.tsx]: Fixed - remove logic from JSX to component

**Pull request merged**